sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Common/EventTriggerMetrics.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
+{
+    /// <summary>
+    /// Static class to set the metric headers for each event trigger.
+    /// </summary>
+    public static class EventTriggerMetrics
+    {
+        /// <summary>
+        /// The client library's product name
+        /// </summary>
+        public static string ProductName = "AuthenticationEvents";
+
+        /// <summary>
+        /// Current executing assembly
+        /// </summary>
+        private static readonly Assembly assembly = typeof(EventTriggerMetrics).Assembly;
```

Comment:
This is not current executing assembly, this is assembly where `EventTriggerMetrics` is defined. Please either change the description or the implementation